docs/manual/using-templates.rst

```diff
@@ -0,0 +1,97 @@
+.. raw:: html
+
+   <div class="new">
+
+New in Tiled 1.1
+
+.. raw:: html
+
+   </div>
+
+Using Templates
+===============
+
+Any created object can be saved with all of its properties as a template, this
+template can be used to create copies of the object on object layers. This can
+save a lot of tedious work of setting up the object type and properties, or
+even just finding the right tile in the tileset.
+
+Templates are organized into template groups. Template groups can be part of
+your project and are referred to by the map. You can save them in either XML
+or JSON format, just like map and tileset files.
+
+.. figure:: images/templates/overview.png
+   :alt: Templates Overview
+
+Creating Templates
+------------------
+
+A template can be create by right clicking on any object in the map and
+selecting "Save as Template". You will be asked to choose the name and the
+template group to save the template to. If the object already has a name the
+name field will be auto-filled with it.
+
+Template groups can be created from the new template group button in the
+templates view or from the new template dialog.
```

Comment:
Capitalization of named things: "Templates view or from the New Template dialog"